README.md

```diff
@@ -1,21 +1,41 @@
+<div align="center">
+  <h1>Unified InstantSearch for Ecommerce</h1>
+  <p><strong>A ready-to-use, fully functional search interface on top of InstantSearch</strong></p>
+</div>
 
-<p align="center">
-<a href="https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/react/"><img src="./.github/unified-instantsearch-ecommerce.png" alt="Unified InstantSearch E-Commerce" width="1280" /></a>
-</p>
+---
 
-**This project aims to be used by any e-commerce website to bootstrap a search experience powered by Algolia the fastest way possible.**
+Unified InstantSearch for Ecommerce combines the most essential widgets and best practices of a great search experience while retaining the customizability of [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/).
 
-While you would still have to push your data to Algolia, the goal of Unified InstantSearch E-Commerce is to remove the need to implement front-end logic and be able to start using Algolia as quickly as possible.
+It only requires a few steps to get up and running with a production-level search solution.
+
+<a href="https://unified-instantsearch-ecommerce.netlify.app" target="_blank">
+  <img src="./.github/unified-instantsearch-ecommerce.png" alt="Unified InstantSearch for Ecommerce" width="1280" />
+</a>
+
+<p align="center">
+  <strong>
+  <a href="https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/react/">Documentation</a> •
+  <a href="https://github.com/algolia/unified-instantsearch-ecommerce/wiki/Configuration-options">Configuration</a> •
+  <a href="https://unified-instantsearch-ecommerce.netlify.app/">Live Demo</a>
+  </strong>
+</p>
 
 ## Documentation
 
-To get started with Unified InstantSearch E-Commerce, head over to the [wiki](https://github.com/algolia/unified-instantsearch-ecommerce/wiki).
+The [documentation](https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/react/) offers a few ways to learn about Unified InstantSearch for Ecommerce:
+
+- Read the [Getting Started](https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/tutorials/getting-started/react/) tutorial to setup your local version of Unified InstantSearch and integrate it into your website
+- Find out how to [Use Your Own Data](https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/tutorials/using-your-own-data/react/) by formatting it specifically for Unified InstantSearch and pushing it to Algolia
+- Learn how to [Customize the UI](https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/tutorials/customizing-your-ui/react/) to make it blend with your website's branding
+- Setup advanced [Merchandising and Rules](https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/tutorials/merchandising-and-rules/react/) to promote items and add banners
 
 ## Disclaimer
```

Comment:
@seafoox I think this might be premature. Unified InstantSearch doesn't have any tests, the API was decided upon only by two members of the team (@francoischalifour and I) and we haven't reviewed it since 2020. Additionally, I don't believe we've made a decision regarding whether we're going to support it or discontinue it in the future, so if we make it beta right now we're committing to some level of support and not changing the API without a major.

I think we should discuss this as a team before, and if we want to move to public beta, take the quality assurance steps required before doing so.

Reply:
Ok, I'll set up time for us to discuss it.

Reply:
@seafoox 👍 Let's make sure to include Youcef and Aayush in the discussion.

Reply:
Aligned, we'll need to wait for Youcef to come back from holiday after easter.